header.php

```diff
@@ -162,9 +162,9 @@
                                 
                                		 <form id="search" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
                                         <span aria-hidden="true" data-icon="&#xf002;"></span>
-                                        <input class="span3" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
+                                        <input role="search" class="span3" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
```

Comment:
This is incorrect use of this ARIA landmark. The role of `search` should be applied on the `<form>` tag, not on the input field. If you would like to note that the input is a search input, you can use `type="search"` instead of the current `type="text"`

Reply:
Resolved in a commit to this branch.